content/systems/filecoin_blockchain/message_pool/message_syncer.md

```diff
@@ -1,34 +1,32 @@
 ---
-title: Message Syncer
+title: Message Propagation
 weight: 1
 dashboardWeight: 2
 dashboardState: incomplete
 dashboardAudit: 0
 dashboardTests: 0
 ---
 
-# Message Syncer
+# Message Propagation
 ---
 
-Transaction messages are sorted in the `mpool` of miners as they arrive according to cryptoeconomic rules followed by the miner.
+The message pool has to interface with the libp2p pubsub [GossipSub](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub) protocol. This is because transaction messages are propagated over [GossipSub](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub) _topics_. Every serialised `SignedMessage` is announced (see [Message](message)) in the corresponding topic by any node participating in the network.
 
-Block messages are ordered as they arrive at miner nodes in order to extend the blockchain.
-
-
-## Message Propagation
-
-Messages are propagated over libp2p pubsub topics, using the [GossipSub](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub) protocol. Every serialised `SignedMessage` is announced (see [Message](message)) in the corresponding topic by any node participating in the network.
-
-There are two pubsub topics related to the mempool: i) the `/fil/msgs/` topic that carries transaction messages and, ii) the `/fil/blocks/` topic that carries block messages. The process is as follows:
+There are two main pubsub topics related to transaction and block messages: i) the `/fil/msgs/` topic that carries transaction messages and, ii) the `/fil/blocks/` topic that carries block messages. The `/fil/msgs/` topic is linked to the `mpool`. The process is as follows:
```

Comment:
```suggestion
There are two main pubsub topics related to transactions and blocks: i) the `/fil/msgs/` topic that carries transactions and, ii) the `/fil/blocks/` topic that carries blocks. The `/fil/msgs/` topic is linked to the `mpool`. The process is as follows:
```
I assume throughout we are talking about Filecoin (not libp2p) messages. Otherwise if we are referring to the two types of message in the same sentence without proper qualification it can get rather confusing. This suggested edit goes in the line of simplification by removing the mention of libp2p messages, we send Filecoin blocks and messages over the network (without the container "message" noun, how libp2p sees them is beyond this spec).